pysph/solver/application.py

```diff
@@ -1236,6 +1237,24 @@ def _write_info(self, filename, **kw):
         info.update(kw)
         json.dump(info, open(filename, 'w'))
 
+    def _log_solver_info(self, solver):
+        sep = '-'*70
+        eqn_info = '[\n' + ',\n'.join([str(e) for e in self.equations]) + '\n]'
+        particle_info = '\n  '.join(['%s: %d' % (p.name, len(p.gid)) for p in
+                                    solver.particles])
+
+        kernel_name = solver.kernel.__class__.__name__
+        kernel_info = '%s(dim=%s)' % (kernel_name, solver.dim)
+
+        logger.info(
```

Comment:
Can use `self._message` for the particle information alone as it is often useful to see.  Also please provide information on the NNPS object used.

Reply:
updated with nnps info, thanks!